gradle/libs.versions.toml

```diff
@@ -1,52 +1,52 @@
 [versions]
 kotlin-version = "1.8.22"
-kotlinx-html-version = "0.9.1"
+kotlinx-html-version = "0.11.0"
 coroutines-version = "1.7.1"
 atomicfu-version = "0.19.0"
-serialization-version = "1.5.1"
+serialization-version = "1.6.2"
```

Comment:
I do believe this will cause problems for publishing the artifacts due to a weird native compilation problem.

https://github.com/ktorio/ktor/pull/3971